models/annotations_models/train_random_forest_classifier_model.ipynb

```diff
@@ -28,49 +28,116 @@
+    "\n",
+    "        annotated_pixels_filepath = data_dir+\"annotaties_coepelduynen_to_pixel.csv\"\n",
+    "        df = pd.read_parquet(annotated_pixels_filepath)\n",
+    "\n",
+    "   \n",
+    "elif location == \"Schippersgat\":\n",
+    "    if satellite_constellation == \"PNEO\":\n",
+    "        if not os.path.isfile(data_dir+\"PNEO_waterplanten_annotations.parquet\"):\n",
+    "            functions.download_file(\"https://e34a505986aa74678a5a0e0f.blob.core.windows.net/satellite-images-nso/Schippersgat/PNEO_waterplanten_annotations.parquet\", data_dir+\"PNEO_waterplanten_annotations.parquet\")\n",
+    "        annotated_pixels_filepath = data_dir+\"PNEO_waterplanten_annotations_schippersgat.parquet\"\n",
+    "        df = pd.read_parquet(annotated_pixels_filepath)\n",
+    "\n",
+    "    if satellite_constellation == \"Superview\":\n",
+    "        if not os.path.isfile(data_dir+\"Superview_waterplanten_annotations.parquet\"):\n",
+    "            functions.download_file(\"https://e34a505986aa74678a5a0e0f.blob.core.windows.net/satellite-images-nso/Schippersgat/Superview_waterplanten_annotations.parquet\", data_dir+\"Superview_waterplanten_annotations.parquet\")\n",
+    "        annotated_pixels_filepath = data_dir+\"Superview_waterplanten_annotations.parquet\"\n",
+    "        df = pd.read_parquet(annotated_pixels_filepath)\n",
+    "\n",
+    "elif location == \"Nieuwkoopse_plassen\":\n",
+    "    if satellite_constellation == \"PNEO\":\n",
+    "        if not os.path.isfile(data_dir+\"PNEO_waterplanten_annotations.parquet\"):\n",
+    "            functions.download_file(\"https://e34a505986aa74678a5a0e0f.blob.core.windows.net/satellite-images-nso/Schippersgat/PNEO_waterplanten_annotations.parquet\", data_dir+\"PNEO_waterplanten_annotations.parquet\")\n",
+    "        annotated_pixels_filepath = data_dir+\"PNEO_waterplanten_annotations_schippersgat.parquet\"\n",
+    "        df = pd.read_parquet(annotated_pixels_filepath)\n",
+    "\n",
+    "\n",
+    "        df = pd.concat([df, pd.read_parquet(data_dir+\"PNEO_waterplanten_annotations_Nieuwkoopse_Plassen.parquet\")])\n",
+    "\n",
+    "    if satellite_constellation == \"Superview\":\n",
+    "        \n",
+    "        if not os.path.isfile(data_dir+\"Superview_waterplanten_annotations.parquet\"):\n",
+    "            print(\"Downloading file\")\n",
+    "            functions.download_file(\"https://e34a505986aa74678a5a0e0f.blob.core.windows.net/satellite-images-nso/Nieuwkoopse_Plassen/Superview_waterplanten_annotations.parquet\", data_dir+\"Superview_waterplanten_annotations.parquet\")\n",
+    "        annotated_pixels_filepath = data_dir+\"Superview_waterplanten_annotations.parquet\"\n",
+    "        df = pd.read_parquet(annotated_pixels_filepath)\n",
+    "\n",
+    "        #TODO: VARY STRANGE REASON THAT NDWI AND NDVI GET FLIPPED WHILE READING IN DATA!!!!!!!!!!!!!!!!!!!!\n",
+    "        df['temp_ndwi'] = df['ndvi']\n",
```

Comment:
Wouldn't it be easier to rename the columns like so:

`df = df.rename(columns={'ndvi': 'ndwi', 'ndwi': 'ndvi'})'`

Reply:
fixed